src/app/components/hmcts-global-header/hmcts-global-header.component.ts

```diff
@@ -14,35 +14,33 @@ import { UserService } from '../../services/user/user.service';
     templateUrl: './hmcts-global-header.component.html',
     styleUrls: ['./hmcts-global-header.component.scss']
 })
-export class HmctsGlobalHeaderComponent implements OnChanges {
+export class HmctsGlobalHeaderComponent implements OnInit, OnChanges {
 
   @Input() public set showNavItems(value: boolean) {
     this.showItems = value;
   }
-  @Input() public items: NavItemsModel[];
+  @Input() public items: NavigationItem[];
   @Input() public logoIsUsed: boolean;
-  @Input() public showFindCase: boolean;
-  @Input() public showCaseReferenceSearchBox: boolean;
   @Input() public headerTitle: {name: string; url: string};
   @Input() public navigation: UserNavModel;
   @Input() public logoType: string;
   @Input() public currentUrl: string;
   @Input() public decorate16DigitCaseReferenceSearchBoxInHeader: boolean;
   @Output() public navigate = new EventEmitter<string>();
 
-  public showItems: boolean;
+  public showItems = true;
   public userValue = true;
   public tab;
-  public get leftItems(): Observable<NavItemsModel[]> {
+  public get leftItems(): Observable<NavigationItem[]> {
     return this.menuItems.left.asObservable();
-  }
-  public get rightItems(): Observable<NavItemsModel[]> {
+  };
+  public get rightItems(): Observable<NavigationItem[]> {
     return this.menuItems.right.asObservable();
   }
 
-  private readonly menuItems = {
-    left: new BehaviorSubject<NavItemsModel[]>([]),
-    right: new BehaviorSubject<NavItemsModel[]>([])
+  private menuItems = {
```

Comment:
@johnbenjamin-hmcts - this should be marked as `readonly` (as before).

Reply:
Thank you @Daniel-Lam - I have now reverted the changes.